Parsers/ASimWebSession/Parsers/ASimWebSessionBarracudaWAF.yaml

```diff
@@ -0,0 +1,306 @@
+          DstBytes = tolong(ReceivedBytes),
+          DstPortNumber = toint(coalesce(DestinationPort,FieldDeviceCustomNumber1)),
+          HttpCookie = RequestCookies,
+          HttpReferrer = RequestContext,
+          HttpRequestBodyBytes = tolong(ReceivedBytes),
+          HttpRequestMethod = RequestMethod,
+          HttpResponseBodyBytes = tolong(SentBytes),
+          NetworkDuration = toint(FlexNumber2),
+          HttpUserAgent = RequestClientApplication,
+          NetworkSessionId = SourceUserID,
+          Rule = RuleName,
+          SrcPortNumber = toint(SourcePort),
+          SrcUsername = SourceUserName,
+          DstUsername = DestinationUserName,
+          Url = RequestURL,
+          HttpResponseCacheControl = iff(
+                                FieldDeviceCustomNumber2 == 0,
+                                "Response from the server",
+                                "Response from the cache"
+                            ),
+          AdditionalFields = bag_pack(
+                        "ProxyIP",
+                        iff(DeviceEventCategory == "WF", DeviceCustomString5, DeviceCustomString3),
+                        "ProxyPort",
+                        FieldDeviceCustomNumber3
+                    ),
+          DvcHostname = DeviceName,
+          DvcIpAddr = DeviceAddress,
+          EventResultDetails = EventOutcome,
+          HttpVersion = FlexString1
+      | extend 
+          SrcUsernameType = iff(isnotempty(SrcUsername), "Simple", ""),
+          DstUsernameType = iff(isnotempty(DstUsername), "Simple", "")
+      | extend
+          Duration = NetworkDuration,
+          SessionId = NetworkSessionId,
+          EventEndTime = EventStartTime,
+          UserAgent = HttpUserAgent,
+          User = SrcUsername,
+          IpAddr = SrcIpAddr,
+          Src = SrcIpAddr,
+          HttpStatusCode = EventResultDetails
+      | project-away
+          ThreatConfidence,
+          CommunicationDirection,
+          AdditionalExtensions,
+          Device*,
+          Source*,
+          Destination*,
+          Activity,
+          LogSeverity,
+          ApplicationProtocol,
+          ProcessID,
+          ExtID,
+          Protocol,
+          Reason,
+          ReceiptTime,
+          SimplifiedDeviceAction,
+          OriginalLogSeverity,
+          ProcessName,
+          EndTime,
+          ExternalID,
+          File*,
+          ReceivedBytes,
+          Message,
+          Old*,
+          EventOutcome,
+          Request*,
+          StartTime,
+          Field*,
+          Flex*,
+          Remote*,
+          Malicious*,
+          severity,
+          ThreatSeverity,
+          IndicatorThreatType,
+          ThreatDescription,
+          _ResourceId,
+          SentBytes,
+          ReportReferenceLink,
+          Computer,
+          EventResult_*,
+          status_code,
+          EventType_lookup,
+          TenantId;
+  union isfuzzy = true 
+      BarracudaCustom,
+      BarracudaCEF
```

Comment:
@jayeshprajapaticrest I'm getting custom table not found when running parser on a workspace that doesn't have custom barracuda table. can you fix this, this is coming for other Barracuda PRs as well.

Reply:
Suggested changes done. Also done same for other schema of barracuda as well.

Reply:
@jayeshprajapaticrest Please re-test on a workspace that doesn't have Barracuda custom table, I'm still getting errors. You can create a new workspace and test
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/cf36f156-c8d9-4cbf-9edc-093b40f4cdea)